src/main/java/de/unistuttgart/moorhuhnbackend/MoorhuhnServiceConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package de.unistuttgart.moorhuhnbackend;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class MoorhuhnServiceConfiguration implements WebMvcConfigurer {
+
+  @Bean
+  public WebMvcConfigurer corsConfigurer() {
+    return new WebMvcConfigurer() {
```

Comment:
These few lines (and the closing parentheses) can be deleted, and then everything should work as intended.